packages/std/src/math/uint512.rs

```diff
@@ -99,6 +99,22 @@ impl Uint512 {
         Uint512(U512::from_little_endian(&value))
     }
 
+    pub const fn from_uint256(num: Uint256) -> Self {
+        let uint256_words = num.to_words();
+        let words: [u64; 8] = [
+            uint256_words[0],
+            uint256_words[1],
+            uint256_words[2],
+            uint256_words[3],
+            0,
+            0,
+            0,
+            0,
+        ];
+
+        Self(U512(words))
```

Comment:
And with https://github.com/CosmWasm/cosmwasm/issues/1111 we can do the same here too, right?

Reply:
I wonder if that would introduce some overhead though when this function isn't used in a `const` context. Does it matter? Probably not.